### DISCUSSION TITLE
Org Settings: Passphrase not enforcing capital/number requirement

### DISCUSSION THREAD

**Discussion by apkehoe** [dani-garcia/vaultwarden#4335] [Q&A]
### Subject of the issue
In organisation settings, I have enforced passphrase requirements:

![image](https://github.com/dani-garcia/vaultwarden/assets/16699289/6f72d058-141c-422b-8723-cac71cf83366)

In both the desktop and browser apps, these enforced settings are not taking effect, specifically for Capitalise and Include Number
Browser:
![image](https://github.com/dani-garcia/vaultwarden/assets/16699289/a31330b4-93c2-42ae-8acf-98ad8ec4cc9d)

Desktop:
![image](https://github.com/dani-garcia/vaultwarden/assets/16699289/04d27976-506f-4f64-80ef-7c413df93bd3)

### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.30.3
* Web-vault version: v2024.1.2
* OS/Arch: linux/x86_64
* Running within Docker: true (Base: Debian)
* Environment settings overridden: false
* Uses a reverse proxy: false
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Browser/Server Time Check: true
* Server/NTP Time Check: true
* Domain Configuration Check: false
* HTTPS Check: false
* Database type: SQLite
* Database version: 3.44.0
* Clients used: 
* Reverse proxy and version: 
* Other relevant information: 

### Config (Generated via diagnostics page)
<details><summary>Show Running Config</summary>

* vaultwarden version: 1.30.3

* Install method: Docker

* Clients used:  Web Vault, Desktop, Browser

* Reverse proxy and version: nginx v1.24.0

* MySQL/MariaDB or PostgreSQL version: N/A

* Other relevant details:

### Steps to reproduce
Enforce passphrase settings in web vault org settings, sync clients, settings not enforced

### Expected behaviour
If I tick the Capitalise/Number boxes, it would select those items and force the behaviour

### Actual behaviour
The web/desktop/browser does not see the settings as ticked so don't enforce the setting.

**Comment by stefan0xC** [comment]
Duplicate of https://github.com/dani-garcia/vaultwarden/issues/4298

Also it's a bug in the client and not something we can fix cf. https://github.com/dani-garcia/vaultwarden/issues/4298#issuecomment-1920041364

**Reply by apkehoe** [reply]
Ah.. I searched for "passphrase" not password. That's why I didn't see the existing item.
